Week3Project/Week3Project/users.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Week3Project
+{
+    class User
+    {
+        public string firstName { get; set; }
+        public string lastName { get; set; }
+        public int favoriteNumber { get; set; }
+        public int numberOfPets { get; set; }
+        public string namesOfPets { get; set; }
+
+        public string AnswerQuestion(string question)
+        {
+            Console.WriteLine(question);
+            return Console.ReadLine();
+
+        }
+        public int AnswerQuestionWithInteger(string question)
+        {
+            string stringAnswer = AnswerQuestion(question);
+            return int.Parse(stringAnswer);
```

Comment:
This works, and it's good to get it working then come back later to improve. Could you improve this line/method so that it doesn't throw an exception if the user doesn't type an integer? (perhaps an `int.TryParse` instead of the `int.Parse`)